.github/workflows/site.yml

```diff
@@ -74,7 +74,7 @@ jobs:
     outputs:
       message: ${{ steps.out.outputs.message}}
     steps:
-      - name: Checkout repository from origin 
```

Comment:
How you detected this? Did you have some CI failure ?

Reply:
I deleted the trailing space because it caused a test to fail.

Reply:
I don't see failures in CIs, please share details how failure looks on your side

Reply:
I rebased your code to remove this change from your PR.